config/environments/production.rb

```diff
@@ -17,7 +17,6 @@
   # Full error reports are disabled and caching is turned on.
   config.consider_all_requests_local = false
   config.action_controller.perform_caching = true
-  config.action_controller.asset_host = ENV['CDN_HOST'] if ENV['CDN_HOST'].present?
```

Comment:
This is just moved below to top level config, not removed.

I think this configuration was setting the asset host for controller only but not also for mailers ... and I'm assuming we want it for both (hence move to top level). If that's NOT true, I'll revert.